calkit/cli/overleaf.py

```diff
@@ -216,7 +217,7 @@ def import_publication(
             name=stage_name,
             environment=tex_env_name,
             kind=StageKind.latex,
-            target=os.path.join(dest_dir, tex_path),
+            target=PurePosixPath(dest_dir, tex_path),
```

Comment:
To ensure this is a `str`:

```suggestion
            target=PurePosixPath(dest_dir, tex_path).as_posix(),
```